### PR TITLE
Update bdr.stat_subscription reference

### PR DESCRIPTION
**Pull request by cwcaro**
Add new column descriptions for PGD 5.2 and add undocumented columns from previous releases.

Reorder column descriptions to match query display order.

BDR-3784

### CODE REVIEW

**Comment by cwcaro** [comment]
@djw-m  note the "pre_commit" phrase in descriptions. Don't know if human readable "before commit" is awkward or accurate. I know the least about pre_commit functionality of all the features I described.

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx [hunk 1]
```diff
@@ -881,6 +887,14 @@ is enabled.
 | start_lsn            | pg_lsn                   | LSN from which this subscription requested to start replication from the upstream                                   |
 | retries_at_same_lsn  | bigint                   | Number of attempts the subscription was restarted from the same LSN value                                           |
 | curr_ncommit         | bigint                   | Number of commits this subscription did after the current connection was established                                |
+| npre_commit_confirmations | bigint              | Number of pre_commit confirmations by CAMO partners                                                                 |
```
Comment (mwanner) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:890]: I suspect this is not just CAMO partners, but also for Group Commit and Eager All Node transactions.

Comment (djw-m) [inline product_docs/docs/pgd/5/reference/catalogs-visible.mdx:890]:
```suggestion
| npre_commit_confirmations | bigint              | Number of pre-commit confirmations by CAMO partners                                                                 |
```

Comment (djw-m) [reply product_docs/docs/pgd/5/reference/catalogs-visible.mdx:890]: A quick look at the code makes me suspect CAMO only as the only place this is incremented is in https://github.com/EnterpriseDB/bdr/blob/e75fdbe10f3cd75987e70676706d51be65af7734/pglogical_receiver_pglogical.c#L240

Which apart from being wrapped in a #ifdef HAVE_CAMO also seems to have a lot of checks about this being a CAMO partner.

**Comment by djw-m** [comment]
> If left as is, I imagine you would want "pre-commit" (hyphen vs underscore) anyway

Oh yes :)

File: product_docs/docs/pgd/5/reference/catalogs-visible.mdx (hunk 1 above)
Comment (PJMODOS) [reply product_docs/docs/pgd/5/reference/catalogs-visible.mdx]: It's definitely for group commit too yes (there is no longer such thing as eager all node in 5)

Comment (PJMODOS) [reply product_docs/docs/pgd/5/reference/catalogs-visible.mdx]: @djw-m HAVE_CAMO is flag for whether postgres we are compiling against has CAMO patchset which is required for group commit as well (or generally is required for converting normal commit into 2pc)

Comment (djw-m) [reply product_docs/docs/pgd/5/reference/catalogs-visible.mdx]: If this is counting confirmations from CAMO partners with CAMO, what is it counting confirmations from with GROUP COMMIT? ie

Number of pre-commit confirmations by CAMO partners with CAMO or <who> with GROUP COMMIT.

"Group Nodes"?

**Comment by cwcaro** [comment]
These changes blended by DJW-M in another PR. Closing this PR.

**Pull request closed without merging**